src/json.ts

```diff
@@ -1,4 +1,5 @@
 import { log } from "./deps.ts";
+import { ShellOutput } from "./process/shell-output.ts";
 
 export function parseJsonWithLog<T>(result: string): T {
```

Comment:
d: is this method then actually still needed?

Reply:
Yes, there are a few instances that do not use the ShellOuput object, e.g. reading from a file.